build/build.sh

```diff
@@ -43,3 +45,21 @@ wkhtmltopdf \
   --quiet \
   output/index.html \
   output/manuscript.pdf
+
+# Create DOCX output when user specifies to do so
+if [[ $BUILD_DOCX == true ]]
+then
+    echo "Exporting Word Docx manuscript"
+    pandoc --verbose \
+    --smart \
+    --from=markdown \
+    --to=docx \
+    --filter pandoc-fignos \
+    --filter pandoc-tablenos \
+    --bibliography=$BIBLIOGRAPHY_PATH \
+    --csl=$CSL_PATH \
+    --reference-docx=$DOCX_PATH \
+    --metadata link-citations=true \
```

Comment:
Note that we now have the option to put `link-citations: true` in `metadata.yaml`. Not sure if we should do this... if so we can do this in a separate PR.